content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,187 @@
+
+== Envoy
+
+Read the <<Envoy>> definition in link:https://istio.io/help/glossary/#envoy[Istio Glossary]
+
+== Envoy Health
+
+It's the status of the incoming and outcoming traffic.
+
+== Error Rate
+
+It's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Read the <<Gateway>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
+
+== Istio Config
+
+It's a configuration object in a Istio environment
+
+=== Istio Name
+
+It's the name of the <<Istio Config>>
+
+=== Istio object/configuration Type
+
+It's the type of the <<Istio Config>>, could be <<Gateway>>, <<VirtualService>>, <<DestinationRule>>, <<ServiceEntry>>, <<Rule>>, <<QuotaSpec>> or <<QuotaSpecBinding>>.
+
+== Istio Sidecar
+
+Read the <<Istio Sidecar>> definition in link:https://istio.io/docs/reference/commands/sidecar-injector/[Istio Sidecar].
+
+== Quota
+
+A limited or fixed number or amount of resources.
+
+== QuotaSpec
+
+File that define a <<Quota>>
+
+== QuotaSpecBinding
+
+Quota spec binding is used to bind link:https://istio.io/help/faq/mixer/[mixer] quota resource name to a proxy.
+
+== Label
+
+It's a tag. The client/user can identify a set of objects with this.
+
+An empty link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (that is, one with zero requirements) selects every object in the collection.
+
+A null link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (which is only possible for optional selector fields) selects no objects.
+
+For example, Istio use the <<Label App>> & <<Label Version>> in a <<Workload>> to specify the version and the application.
+
+=== Label App
+
+This represents the app label of an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Labels Requirements]
+
+=== Label Version
+
+This represents the version label of an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Labels Requirements]
+
+== Mixer
+
+Read the <<Mixer>> definition in link:https://istio.io/help/glossary/#mixer[Istio Glossary]
+
+== Namespace
+
+Namespaces are intended for use in environments with many users spread across multiple teams, or projects
+
+Namespaces are a way to divide cluster resources between multiple users
+
+== Outbound Metrics
+
+Outcoming Metrics to a <<Workload>> or <<Service>>
+
+== Pilot
+
+Read the <<Pilot>> definition in link:https://istio.io/help/glossary/#pilot[Istio Pilot]
+
+== Port
+
+Read the <<Port>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Port[Istio Port]
+
+== Rate Interval
+
+It's an amount of time. By Default in Kiali last 10 minutes.
```

Comment:
To be more accurate:

"It's the amount of time by which a rate is computed" ? (is that correct English?)

Also I'm not sure if we want to display here Kiali's default, I'm not sure it's very interesting, and it could change, and we could have different defaults depending on pages in the future.